modules/auxiliary/scanner/mssql/mssql_login.rb

```diff
@@ -35,14 +36,30 @@ def initialize
       OptBool.new('TDSENCRYPTION', [ true, 'Use TLS/SSL for TDS data "Force Encryption"', true]),
     ])
 
-    deregister_options('PASSWORD_SPRAY')
+    options_to_deregister = %w[PASSWORD_SPRAY]
+    unless framework.features.enabled?(Msf::FeatureManager::MSSQL_SESSION_TYPE)
```

Comment:
We're adding in notification messages here now to let users know that the functionality exists, i.e. here's the postgres module code

```
    options_to_deregister = %w[SQL PASSWORD_SPRAY]
    if framework.features.enabled?(Msf::FeatureManager::POSTGRESQL_SESSION_TYPE)
      add_info('New in Metasploit 6.4 - The %grnCreateSession%clr option within this module can open an interactive session')
    else
      options_to_deregister << 'CreateSession'
    end
    deregister_options(*options_to_deregister)
```

Reply:
Looks like this is still needing to be applied 👀